Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -55,8 +54,9 @@ private string Me
 
         #region Grain overrides
 
-        public override Task OnActivateAsync()
+        public override async Task OnActivateAsync()
         {
+            await ReadStateAsync();
```

Comment:
why do u need to explicitly read?  The ReadStateAsync is done automatically before OnActivateAsync.